conda_forge_tick/migrators.xsh

```diff
@@ -336,6 +336,12 @@ class Compiler(Migrator):
     rerender = True
 
     compilers = {'toolchain', 'gcc'}
+    patterns = (('meta.yaml', '  number:\s*[0-9]+', '  number: $BUILDNUM'),
```

Comment:
Thank you!

Reply:
Would it be possible to put this everywhere else? (Not version bump but I think all the others need this too (noarch, JS).